system-test/storage.ts

```diff
@@ -1179,44 +1103,29 @@ describe('storage', () => {
       );
     });
 
-    it('should append a new rule', done => {
+    it('should append a new rule', async () => {
       const numExistingRules =
-        bucket.metadata.lifecycle && bucket.metadata.lifecycle.rule.length;
+        (bucket.metadata.lifecycle && bucket.metadata.lifecycle.rule.length) ||
+        0;
 
-      async.series(
-        [
-          next =>
-            bucket.addLifecycleRule(
-              {
-                action: 'delete',
-                condition: {
-                  age: 30,
-                  isLive: true,
-                },
-              },
-              next
-            ),
-
-          next =>
-            bucket.addLifecycleRule(
-              {
-                action: 'delete',
-                condition: {
-                  age: 60,
-                  isLive: true,
-                },
-              },
-              next
-            ),
-        ],
-        err => {
-          assert.ifError(err);
-          assert.strictEqual(
-            bucket.metadata.lifecycle.rule.length,
-            numExistingRules + 2
-          );
-          done();
-        }
+      await bucket.addLifecycleRule({
+        action: 'delete',
+        condition: {
+          age: 30,
+          isLive: true,
+        },
+      }),
```

Comment:
weird formatting - would expect a semicolon here

Reply:
True, fixed it